app/app.go

```diff
@@ -20,33 +21,35 @@ import (
 )
 
 type App struct {
-	Srv *Server
-
-	Log              *mlog.Logger
-	NotificationsLog *mlog.Logger
-
-	T              goi18n.TranslateFunc
-	Session        model.Session
-	RequestId      string
-	IpAddress      string
-	Path           string
-	UserAgent      string
-	AcceptLanguage string
-
-	AccountMigration einterfaces.AccountMigrationInterface
-	Cluster          einterfaces.ClusterInterface
-	Compliance       einterfaces.ComplianceInterface
-	DataRetention    einterfaces.DataRetentionInterface
-	Elasticsearch    einterfaces.ElasticsearchInterface
-	Ldap             einterfaces.LdapInterface
-	MessageExport    einterfaces.MessageExportInterface
-	Metrics          einterfaces.MetricsInterface
-	Notification     einterfaces.NotificationInterface
-	Saml             einterfaces.SamlInterface
-
-	HTTPService httpservice.HTTPService
-	ImageProxy  *imageproxy.ImageProxy
-	Timezones   *timezones.Timezones
+	srv *Server
+
+	log              *mlog.Logger
+	notificationsLog *mlog.Logger
+
+	t              goi18n.TranslateFunc
+	session        model.Session
+	requestId      string
+	ipAddress      string
+	path           string
+	userAgent      string
+	acceptLanguage string
+
+	accountMigration einterfaces.AccountMigrationInterface
+	cluster          einterfaces.ClusterInterface
+	compliance       einterfaces.ComplianceInterface
+	dataRetention    einterfaces.DataRetentionInterface
+	elasticsearch    einterfaces.ElasticsearchInterface
+	ldap             einterfaces.LdapInterface
+	messageExport    einterfaces.MessageExportInterface
+	metrics          einterfaces.MetricsInterface
+	notification     einterfaces.NotificationInterface
+	saml             einterfaces.SamlInterface
+
+	httpService httpservice.HTTPService
+	imageProxy  *imageproxy.ImageProxy
+	timezones   *timezones.Timezones
+
+	context context.Context
```

Comment:
Do we need `context` here?

Reply:
We do. It's per App, and is used for opentracing